app/views/shared/_search_bar.html.erb

```diff
@@ -1,5 +1,5 @@
-<div id="user-search-bar" class="col-md-5 row pull-right">
```

Comment:
shared/_search_bar is a bit too generic a name, maybe 'shared/_user_search_bar'?

Because there are project search bars as well. Also, leave in the 'user-' prefix for the ID as well, since you're sharing the DOM with other partials.